### ISSUE TITLE
Login Screen Image Link Invalid

### ISSUE THREAD

**Issue by mattdawkins** [Kitware/dive#1434]
Looks like at some point the login image/icon became invalid

https://viame.kitware.com/static/viame/img/logo.22a6f4a3.png

leading to the below

![Screenshot at 2024-07-05 12-40-38](https://github.com/Kitware/dive/assets/2353972/305a5f71-ca92-402b-8030-708e16fd2bd9)

**Comment by BryonLewis** [comment]
This should be resolved.
I completely forgot but our logo is configured using the endpoint for 'brand_data' which allows us to point to any image for the URL src of logo.  I don't remember why we did this.  But I should probably change it so the image is in a public location on github or on the server in a URL location.  It was pointing instead of a built artificat from the client process.

**Comment by waxlamp** [comment]
I assume we did it so that different orgs can custom-brand their DIVE deployments, right? How are we doing that for Terpsi?

**Comment by BryonLewis** [comment]
Yeah we did it for that. I'm just curious as to why it seems to be set to an internal build artifact of the client application that is transferred to the /static/viame folder during the docker build.  It may make more sense as a public GirderItem URL that is referenced instead.

**Comment by waxlamp** [comment]
Oh I see what you mean. Thanks.

**Comment by BryonLewis** [comment]
I created a 'Branding' Collection on the server with the VIAME Developers group (us) given access.  Inside is a Branding Image folder and the logo image is inside.  The image src is now pointed to that file.  I believe that using a client build artifact was just a remnant of before the branding system was implemented.
